### PR TITLE
Web release notes 1901

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -14,6 +14,29 @@ description: Release Notes for 100ms JavaScript SDK
...
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2024-01-19
+Released: `@100mslive/hms-video-store@0.11.1`, `@100mslive/react-sdk@0.9.1`, `@100mslive/hls-player@0.2.1`, `@100mslive/roomkit-react@0.2.1`
+
+
+#### Fixed:
+- Roomkit Prebuilt: Zoom for whiteboard on mweb
+- Roomkit Prebuilt: Participation summary view for voters
+- Roomkit Prebuilt: Certain element borders showing up
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:24]: Echo in firefox versions greater than 116

Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:24]: Missing reset css on some components

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -14,6 +14,30 @@ description: Release Notes for 100ms JavaScript SDK
...
+## 2024-01-12
+Released: `@100mslive/hms-video-store@0.11.0`, `@100mslive/react-sdk@0.9.0`, `@100mslive/hls-player@0.2.0`, `@100mslive/roomkit-react@0.2.0`
+
+#### Added:
+- Roomkit Prebuilt: Chat supports reply to peer and reply to group for group messages
+- Roomkit Prebuilt: Pinned messages display the name of the peer who pinned the message
+- 100ms prebuilt links now accept `userId` param in the URL to set the userId of a peer
+
+#### Fixed:
+- Mismatch of tracks in transceiver
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:37]: Audio not audible to remote peers when you join as muted and then unmute on mweb

File: docs/javascript/v2/release-notes/release-notes.mdx (hunk 1 above)
Comment (KaustubhKumar05) [reply docs/javascript/v2/release-notes/release-notes.mdx]: Updated

**Pull request merged**